src/test/java/org/ballerinax/datamapper/DataMapperPluginTest.java

```diff
@@ -204,83 +212,145 @@ public void testHappyPathMultipleClients() {
     public void tearDown() throws IOException {
         //Cleanup the test projects if they already have generated json files
         try {
-            Files.deleteIfExists(Paths.get("src/test/resources/test1/src/module_test1/resources/Assignee_schema.json"));
-            Files.deleteIfExists(Paths.get("src/test/resources/test1/src/module_test1/resources/Client_functions.json"));
-            Files.deleteIfExists(Paths.get("src/test/resources/test1/src/module_test1/resources/Creator_schema.json"));
-            Files.deleteIfExists(Paths.get("src/test/resources/test1/src/module_test1/resources/Issue_schema.json"));
-            Files.deleteIfExists(Paths.get("src/test/resources/test1/src/module_test1/resources/Label_schema.json"));
+            Files.deleteIfExists(Paths.get("src/test/resources/test1/src/module_test1/resources/" +
+                    "Assignee_schema.json"));
+            Files.deleteIfExists(Paths.get("src/test/resources/test1/src/module_test1/resources/" +
+                    "Client_functions.json"));
+            Files.deleteIfExists(Paths.get("src/test/resources/test1/src/module_test1/resources/" +
+                    "Creator_schema.json"));
+            Files.deleteIfExists(Paths.get("src/test/resources/test1/src/module_test1/resources/" +
+                    "Issue_schema.json"));
+            Files.deleteIfExists(Paths.get("src/test/resources/test1/src/module_test1/resources/" +
+                    "Label_schema.json"));
 
-            Files.deleteIfExists(Paths.get("src/test/resources/test2/src/module_test2/resources/Assignee_schema.json"));
-            Files.deleteIfExists(Paths.get("src/test/resources/test2/src/module_test2/resources/Creator_schema.json"));
-            Files.deleteIfExists(Paths.get("src/test/resources/test2/src/module_test2/resources/Creator_data.json"));
-            Files.deleteIfExists(Paths.get("src/test/resources/test2/src/module_test2/resources/Issue_schema.json"));
-            Files.deleteIfExists(Paths.get("src/test/resources/test2/src/module_test2/resources/Label_schema.json"));
-            Files.deleteIfExists(Paths.get("src/test/resources/test2/src/module_test2/resources/Client_functions.json"));
+            Files.deleteIfExists(Paths.get("src/test/resources/test2/src/module_test2/resources/" +
```

Comment:
Shall we extract a variable for `src/test/resources/test2/src/module_test2/resources/`, then we can get this to a single line.

Reply:
Ok. As you mentioned the reason why I broke each line into two was to maintain the line length of 12 characters. I also thought of keeping the path in a separate variable and concatenating. But then I moved to doing the above change because it does not require additional variable to store the path. Anyway, I think the approach you proposed improves the readability of the code. Hence lets use the approach you mentioned.